### PR TITLE
add: sd-jwt-verifier

### PR DESCRIPTION
**Pull request by ragnika**

### CODE REVIEW

**Review by linasi — APPROVED** [review]
LGTM.
@ragnika what's the plan with the tests?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "sd-jwt",
+  "version": "0.0.1",
+  "description": "",
```
Comment (linasi) [inline package.json:4]: @ragnika add some short description here as well

**Comment by ragnika** [comment]
> LGTM. @ragnika what's the plan with the tests?

working on add tests that read example files generated by the  python script as inputs.
That way we are testing against the examples generated by the ref implementation

**Comment by ragnika** [comment]
> Can you please add more documentation to
> 
> * All public facing methods
> * Some of the more complex methods could also use some inline comments (e.g. unpack)

@javereec comments added

**Review by javereec — APPROVED** [review]
Thanks!

**Pull request merged**